src/main/java/seedu/address/logic/commands/ExitCommand.java

```diff
@@ -8,6 +8,7 @@
  */
 public class ExitCommand extends Command {
 
+    public static final String COMMAND_ALIAS = "e";
```

Comment:
I think its better if you make this ex for clarity between edit and exit.

Reply:
I thought the user would be in a rush while exiting and would want to type less. That's why. Still, want to make it "ex"?

Reply:
Yes.